build/CopyFilesToStagingDir.ps1

```diff
@@ -116,6 +116,7 @@ PublishFile $FullBuildOutput\Microsoft.Windows.ApplicationModel.DynamicDependenc
 PublishFile $FullBuildOutput\Microsoft.Windows.ApplicationModel.DynamicDependency.Projection\Microsoft.Windows.ApplicationModel.DynamicDependency.Projection.pdb $NugetDir\lib\net5.0-windows10.0.17763.0
 PublishFile $FullBuildOutput\Microsoft.Windows.AppLifecycle.Projection\Microsoft.Windows.AppLifecycle.Projection.dll $NugetDir\lib\net5.0-windows10.0.17763.0
 PublishFile $FullBuildOutput\Microsoft.Windows.AppLifecycle.Projection\Microsoft.Windows.AppLifecycle.Projection.pdb $NugetDir\lib\net5.0-windows10.0.17763.0
```

Comment:
@EHO-makai, @Scottj1s, @BenJKuhn ... do we think that long-term we want lots of these DLLs? Or can they be merged?